samples/bluetooth/mesh/light/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+# Application overlay - nrf54l15
+
+CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE=n
+CONFIG_SPI_NOR=n
```

Comment:
I see this is not your change, but why is this disabled only for this specific sample?